src/styles/index.scss

```diff
@@ -1,3 +1,24 @@
+@import './utils/variebles';
```

Comment:
There is a typo in the file path for 'variables'. It should be spelled as 'variables' instead of 'variebles'.